src/styles/index.scss

```diff
@@ -1,3 +1,137 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+$width: 200px;
+
+* {
+  background: $white;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  background: #fff;
+}
+
+.card {
+  display: flex;
+  width: $width;
+  padding: 32px 16px 16px;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  .stars {
+    display: flex;
+
+    &__star {
+      width: 16px;
+      height: 16px;
+      margin-right: 4px;
+      background-image: url(../images/star.svg);
+      background-position: center ;
+      background-repeat: no-repeat;
+    }
+
+    &--4 .stars__star {
+      &:nth-child(-n+4) {
+        background-image: url(../images/star-active.svg);
+      }
+    }
+  }
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-accent;
+  }
+
+  &__code {
+    margin: 0;
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+  }
+
+  &__reviews {
+    display: flex;
+    margin-bottom: 24px;
+    align-items: flex-end;
+    justify-content: space-between;
+
+    &-text {
+      margin: 0;
+      padding: 0;
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
```

Comment:
it's default value, remove

Reply:
Hello. After I made the changes you indicated, not one of the tests fails. Write what I should do in such a situation.